doc/eventhandler/requestmangler.rst

```diff
@@ -0,0 +1,90 @@
+.. _requestmanglerhandler:
+
+RequestMangler Handler Module
+-----------------------------
+
+.. index:: RequestMangler, Handler Modules
+
+The RequestMangler is a special handler module, that can modify
+the request parameters of an HTTP request.
+This way privacyIDEA can change the data that is processed within the request.
+
+Usually this handler is used in the **pre** location. However there might be occasions
+when you want to modify paramters only *before* passing them to the next handler. In this case
+you can also use the RequestMangler handler in the **post** location.
```

Comment:
If I understand it correctly, having a **post** RequestMangler handler makes sense if one wants to modify parameters only before passing them to the next **post** event handler. Maybe we should make this clearer by writing "passing them to the next **post** handler" above?

Reply:
done in a8f1447c